src/views/PlaygroundView.tsx

```diff
@@ -9,12 +10,12 @@ export const PlaygroundView = () => {
     <Container>
       <Text variant="h2">Internal testing view</Text>
       <LinksContainer>
-        <Link to="./first">First</Link>
+        <Link to="./form">Form Validation</Link>
         <Link to="./second">Second</Link>
         <Link to="./third">Third</Link>
       </LinksContainer>
       <Routes>
-        <Route key="first" path="/first" element={<p>First</p>} />
+        <Route key="form" path="/form" element={<PlaygroundForm />} />
```

Comment:
Should we name that differently? Maybe `PlaygroundValidationForm` or `PlaygroundValidation`?
Not sure how this playground will look in the future, but I think there will be more `form` components there.